src/CommunityHydration.js

```diff
@@ -0,0 +1,46 @@
+// const Hydration = require('../src/Hydration');
+
+class CommunityHydration {
+  constructor(data = []) {
+    this.hydrations = data.map(user => new Hydration(user))
+  }
+
+  calculateAvgWaterPerDay(userID) {
+    const userWater = this.hydrations.filter(water => (water.userID === userID));
+    const waterConsumed = userWater.map(water => water.numOunces);
+    const avgWaterConsumed = (waterConsumed.reduce((a, b) => a + b, 0)) / waterConsumed.length;
+    return avgWaterConsumed;
+  }
+
+  calculateTotalWaterOnDay(userID, date) {
+    const userWater = this.hydrations.find(water => water.userID === userID && water.date === date);
+    return userWater.numOunces;
+  }
+
+  convertDateString(date) {
+    return parseInt(date.split('/').join(''));
+  }
+
```

Comment:
This looks nice, I appreciate the clear names. We might want to talk to a mentor to help decide on ways to refactor or pull out smaller helper functions to make this a less scary big block of code.